pkg/plugins/policies/meshloadbalancingstrategy/plugin/v1alpha1/plugin.go

```diff
@@ -94,40 +99,61 @@ func (p plugin) configureDPP(
 	// we configure clusters in a separate loop to avoid configuring the same cluster twice
 	for serviceName, conf := range serviceConfs {
 		if cluster, ok := clusters.Outbound[serviceName]; ok {
-			if err := p.configureCluster(cluster, conf.LoadBalancer); err != nil {
+			if err := p.configureCluster(cluster, conf); err != nil {
 				return err
 			}
+			if cluster.LoadAssignment != nil {
+				if err := ConfigureStaticEndpointsLocalityAware(proxy, endpoints, splitEndpoints, cluster, conf, serviceName); err != nil {
+					return err
```

Comment:
Please add context to every error in this function (and file maybe) so if something fails, we will know which cluster failed